### PR TITLE
remove code for JavaScript Error Logging

### PR DESCRIPTION
**Pull request by owend**
- Feature was already deprecated/disabled
- This PR just removes the unused code

@macbre @ljagiello

### CODE REVIEW

File: lib/Wikia/src/Logger/Hooks.php [hunk 1]
```diff
@@ -12,72 +12,6 @@
...
-
-		if ( !$wgDevelEnvironment ) {
-			$onError = $wgIsGASpecialWiki || $wgEnableJavaScriptErrorLogging;
-			$key = "wikialogger-top-script-$onError";
-			$loggingJs = $wgMemc->get( $key );
-
-			if ( !$loggingJs ) {
-				$errorUrl = "//jserrorslog.wikia.com/";
-				$loggingJs = "
-					function syslogReport(priority, message, context) {
```
Comment (macbre) [inline lib/Wikia/src/Logger/Hooks.php]: This function is called in `resources/wikia/modules/log.js`. The call is "safe" as it's wrapped in an if statement, but let's remove it :)

Comment (owend) [inline lib/Wikia/src/Logger/Hooks.php]: Oh yeah, there is actually quite a lot of code that references it... I will remove those references too.

**Comment by owend** [comment]
There is a function called veTrack() which is defined in tracker.stub.js and used quite a lot in the VE.  It also calls syslogReport.  I am contacting that team about the best approach to take there.  @inez

**Comment by hakubo** [comment]
lgtm

**Pull request merged**